### PR TITLE
[12.0][FIX] hr_timesheet_sheet: don't store possible_reviewer_ids

### PR DESCRIPTION
**Pull request by alexey-pelykh**
Right now set of possible reviewers is frozen at moment of timesheet sheet creation

### CODE REVIEW

File: hr_timesheet_sheet/models/hr_timesheet_sheet.py [hunk 1]
```diff
@@ -246,24 +240,36 @@ def _compute_total_time(self):
...
+            check = check_in
         else:
-            operator = '!='
-        return [('possible_reviewer_ids', operator, self.env.uid)]
+            check = check_not_in
+
+        sheets = self.browse().filtered(
+            lambda sheet: check(sheet._get_possible_reviewers())
+        )
+        _logger.info('!!! %s %s => %s %s %s' % (
```
Comment (leemannd) [inline hr_timesheet_sheet/models/hr_timesheet_sheet.py]: Seems a leftover of debug

Comment (alexey-pelykh) [reply hr_timesheet_sheet/models/hr_timesheet_sheet.py]: Yep, thanks!

**Review by leemannd — APPROVED** [review]
Quick review. Seems legit

**Comment by alexey-pelykh** [comment]
Fast-tracking as fix

**Pull request merged**